wordpress-mobile/AztecEditor-Android
# Add AztecText method to set preformatted background alpha color from the child

### Fix 

This PR adds a method that can change preformatted background alpha color from the child class.

### Test
Test 1:
1. Run AztecEditor Demo app and check that background color isn't change

Test 2:
1. Run Gutenberg mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1442/
where value is set to 0, and see that background is transparent.

### Review
@daniloercoli 

Make sure strings will be translated:
